src/components/SideNavigation/NavigationItem/index.tsx

```diff
@@ -29,6 +29,9 @@ const NavigationItem = (props: TNavigationLinkPropTypes) => {
         return As();
     }
 
+    const displayHeader = () => isOpen && type === NavigationItemTypes.BLOCK_HEADER && showAction && actionElm && actionElm;
```

Comment:
why function?

Reply:
@valodyapapikyan suggested me in the comments, create functions for not to write more ternary operator

Reply:
I mean you can just write     const displayHeader = isOpen && type === NavigationItemTypes.BLOCK_HEADER && showAction && actionElm && actionElm;

and use displayHeader without calling.
Also `actionElm` is checked twise.

Reply:
@LilitArz done